config/configuration.toml.in

```diff
@@ -10,6 +10,13 @@ machine_type = "@MACHINETYPE@"
 # trouble running pre-2.15 glibc
 kernel_params = "@KERNELPARAMS@"
 
+# Path to the firmware.
+# If you want that qemu uses the default firmware leave this option empty
+firmware = "@FIRMWAREPATH@"
+
+# Machine accelerators
+machine_accelerators="@MACHINEACCELERATORS@"
```

Comment:
What format is this variable? Can you add a comment explaining if the accelerators should be space/comma/whatever-separated, ideally with an example showing >1 accelerator being specified.

Reply:
sure, thanks :)